test/integration/targets/loop_control/runme.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# user output has:
+#ok: [localhost] => (item=looped_var foo_label) => {
+#ok: [localhost] => (item=looped_var bar_label) => {
+MATCH='foo_label
+bar_label'
+[ "$(ansible-playbook label.yml "$@" |grep 'item='|sed -e 's/^.*(item=looped_var \(.*\)).*$/\1/')" == "${MATCH}" ]
```

Comment:
Wouldn't assertion statements in the integration test playbook be better/cleaner and easier to maintain over time?

Reply:
maybe, but see comment above